packages/dialog/src/vaadin-dialog-draggable-mixin.js

```diff
@@ -93,10 +93,8 @@ export const DialogDraggableMixin = (superClass) =>
           window.addEventListener('touchend', this._stopDrag);
           window.addEventListener('mousemove', this._drag);
           window.addEventListener('touchmove', this._drag);
-          if (this.$.overlay.$.overlay.style.position !== 'absolute') {
-            const { top, left } = this._originalBounds;
-            this.$.overlay.setBounds({ top, left });
```

Comment:
Actually, we should preserve this behavior with only setting `top` and `left` - otherwise we'll face https://github.com/vaadin/web-components/issues/436 again (it was fixed by https://github.com/vaadin/web-components/pull/7970).

I'll update my PR.

Reply:
Can we consider just fixing the width? I assume the issue in #436 is about the height (“I have a dialog with a details area that I can show and hide”). If we don't fix the width, then we’ll have this weird behavior: 

https://github.com/user-attachments/assets/0406c2cb-a0b4-4344-bd3b-e6168b8a2744

Reply:
One solution to explore is to set `width: max-content` on `[part='overlay']`.

Reply:
<img width="505" alt="Screenshot 2025-06-18 at 8 33 36" src="https://github.com/user-attachments/assets/40e574cb-816e-43c4-bde8-3ab8b4c2ad48" />

Reply:
Will this also solve https://github.com/vaadin/web-components/issues/8790? If so, let's create a separate PR to first apply this fix to existing core styles.

Reply:
Yeah, this should affect that, assuming we revert draggable-mixin behavior back to what it was before (not fixing width/height and not adding ´[has-bounds-set]`).

Reply:
Done

Reply:
Thanks. I also restored corresponding tests.